src/main/java/frc/robot/commands/SysIDDriveCommand.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+
+import com.ctre.phoenix6.SignalLogger;
+import com.ctre.phoenix6.controls.PositionDutyCycle;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Config;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
+import frc.robot.constants.swerve.DriveConstants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.SysId;
+
+/** Add your docs here. */
+public class SysIDDriveCommand extends SequentialCommandGroup {
+
+    Drivetrain drive;
+    Config config = new Config();
+    SysId sysId;
```

Comment:
Make instance variables private instead of the default (package access).

Reply:
done